xbmc/dbwrappers/mysqldataset.cpp

```diff
@@ -176,8 +176,22 @@ int MysqlDatabase::connect(bool create_new) {
       static bool showed_ver_info = false;
       if (!showed_ver_info)
       {
-        CLog::Log(LOGINFO, "MYSQL: Connected to version %s", mysql_get_server_info(conn));
+        std::string version_string = mysql_get_server_info(conn);
+        CLog::Log(LOGINFO, "MYSQL: Connected to version {}", version_string);
```

Comment:
Maybe this could be LOGNOTICE so that it is logged even when debug is not enabled?

Reply:
fine by me